chromedriver.sh

```diff
@@ -2,7 +2,15 @@
 set -ex
 
 CHROMEDRIVER=chromedriver_linux64.zip
-curl -Lo $CHROMEDRIVER http://chromedriver.storage.googleapis.com/${CHROMEDRIVER_VERSION:-2.43}/$CHROMEDRIVER
+
+# See http://chromedriver.chromium.org/downloads/version-selection.
+if ! which google-chrome; then
+  echo "Must have google-chrome installed to determine default version of chromedriver to install."
+  exit 1
+fi
+DEFAULT_CHROMEDRIVER_VERSION=$(google-chrome --version | grep -oE "[0-9]+.[0-9]+.[0-9]+")
+
+curl -Lo $CHROMEDRIVER http://chromedriver.storage.googleapis.com/${CHROMEDRIVER_VERSION:-DEFAULT_CHROMEDRIVER_VERSION}/$CHROMEDRIVER
```

Comment:
you need a `$` in front of `DEFAULT_CHROMEDRIVER_VERSION` (or you get the string back)

Reply:
seems like you're missing a step -- curl'ing the pointer file.

```
# get the chrome patch version
chrome_version=$(google-chrome --version | sed 's/[^0-9.]//g' | cut -d. -f1-3)
# curl the chromedriver pointer
chromedriver_version=$(curl -ssL "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$chrome_version")
# curl chromedriver
curl -Lo $CHROMEDRIVER "http://chromedriver.storage.googleapis.com/$chromedriver_version/$CHROMEDRIVER"
```

Reply:
sorry, should've made this a draft PR... was still iterating on this and testing it in a repo :)